Flutter/cartesian-charts/tooltip.md

```diff
@@ -119,6 +119,7 @@ By default, x and y value will be displayed in the tooltip, and it can be custom
 * Y value - `point.y`
 * Bubble size - `point.size`
 * Name of the series - `series.name`
+* Stores the cumulative value - `point.cumualtive `
```

Comment:
Stores the cumulative value for stacked series

Reply:
resolved the above concerns